dashboard/pkg/epinio/edit/services.vue

```diff
@@ -75,6 +89,10 @@ export default Vue.extend<Data, EpinioCompRecord, EpinioCompRecord, EpinioCompRe
         return false;
       }
 
+      if (!Object.values(this.validChartValues).every((v) => !!v)) {
```

Comment:
just checking, is there no scenario where a falsey is allowed, for instance an empty string?

Reply:
No scenarios, also checked Applications charts